plugins/core/IO/qRDBIO/CMakeLists.txt

```diff
@@ -57,4 +57,39 @@ if( PLUGIN_IO_QRDB )
 	add_subdirectory( ui )
 
 	target_link_libraries( ${PROJECT_NAME} rdbcpp )
+
+	option( PLUGIN_IO_QRDB_INSTALL_DEPENDENCY "Install RDB shared library to library folder" OFF)
+	if (PLUGIN_IO_QRDB_INSTALL_DEPENDENCY)
+		get_target_property(RDB_LIBRARY_FILE rdbc IMPORTED_LOCATION)
+		# resolve possible symlink and install the real library
+		get_filename_component(lib_REAL ${RDB_LIBRARY_FILE} REALPATH)
+		install(FILES ${lib_REAL} DESTINATION lib)
+		# check if given library is no symlink to prevent double installation
+		STRING(COMPARE NOTEQUAL "${RDB_LIBRARY_FILE}" "${lib_REAL}" real_lib_different)
+		# find all the symlinks linking to lib_REAL
+		string(REGEX MATCH "\\.so" is_shared_lib_linux "${RDB_LIBRARY_FILE}")
```

Comment:
why not relying on "if ( UNIX / WIN32 / APPLE )" tests? Or maybe simply add these tests before entering this part of the script?

Reply:
good catch, the check `real_lib_different` can go down to the line 88 with `else() #neither linux nor darwin` line

just to be extra careful so that the later regex `string(REGEX REPLACE "\\.so.*" ".so" lib_base "${RDB_LIBRARY_FILE}")` is sure to be working. But if this is confusing I can switch to `UNIX/WIN32/APPLE` and just assume the file ending based on the OS

Reply:
Oh I like existing code, it's more that I wouldn't bother doing all these on a WIN32 target typically. At least it's easier for me (a Windows user) to know that I don't need to bother with this part of the script ;)

Reply:
Added `NOT WIN32` guard :)